apps/new-tab/src/Settings.tsx

```
@@ -126,10 +182,83 @@ function SettingsTrigger({
   const [opacity, setOpacity] = createStoredSignal<number>("opacity", 0.8);
   const [settingsMenu, setSettingsMenu] = createSignal<string>("general");
   const [dialogOpen, setDialogOpen] = createSignal(false);
+  const [imperial, setImperial] = createStoredSignal("imperial", false);
+  const [city, setCity] = createStoredSignal("locationCity", "");
+  const [bookmarksShown, setBookmarksShown] = createStoredSignal<
+    Array<Bookmark> | string
+  >("bookmarksShown", []);
+  const [clearDataDialogOpen, setClearDataDialogOpen] = createSignal(false);
+  const [pomodoroConfig, setPomodoroConfig] = createStoredSignal<
+    Function | PomodoroConfig | string
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)